p2p/security/noise/transport.go

```diff
@@ -65,3 +83,52 @@ func (t *Transport) WithSessionOptions(opts ...SessionOption) (sec.SecureTranspo
 	}
 	return st, nil
 }
+
+func matchMuxers(initiatorMuxers, responderMuxers []string) string {
+	selectedMuxer := ""
+	for _, muxer := range responderMuxers {
+		for _, initMuxer := range initiatorMuxers {
+			if initMuxer == muxer {
+				selectedMuxer = muxer
```

Comment:
Why not return here?

Reply:
yes, it can return, it will be simpler to return, thanks!